engines/bops_api/app/controllers/bops_api/v2/planning_applications/neighbour_responses_controller.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module BopsApi
+  module V2
+    module PlanningApplications
+      class NeighbourResponsesController < AuthenticatedController
+        def create
+          @planning_application = find_planning_application(params[:planning_application_id])
+          @consultation = @planning_application.consultation
+          neighbour_response = build_neighbour_response
+
+          neighbour_response.save!
+
+          render json: {
+            data: nil,
+            status: {
+              code: 200,
+              message: "OK",
+              detail: "Neighbour response created successfully"
+            }
+          }, status: :ok
+        rescue ActiveRecord::RecordInvalid
```

Comment:
We handle `ActiveRecord::RecordInvalid` in `BopsApi::ErrorHandler` - is there a reason why you're rescuing it here?

Reply:
ah this one might be my fault I may have said to add it in too soon! 

We will need to make some sort of Postsubmission response handler so we can implement the [ApiResponse structure](https://github.com/theopensystemslab/digital-planning-data-schemas/blob/main/IMPLEMENTATION.md#api-response-structure) for the ODP public endpoints. 

For now we can use your [previous comment](https://github.com/unboxed/bops/pull/2403/files#r2195003597) and let `BopsApi::ErrorHandler` handle it for now. 

I'll make a ticket in our backlog for the handler, @pixeltrix do you have any recommendations for how best to approach it?